README.md

```diff
@@ -44,9 +44,11 @@ Outpox
 [PokemonGo-DesktopMap](https://github.com/mchristopher/PokemonGo-DesktopMap) Amazing job done by  Mike Christopher
 
 ## Licensing
+©2016 Niantic, Inc. ©2016 Pokémon. ©1995–2016 Nintendo / Creatures Inc. / GAME FREAK inc. © 2016 Pokémon/Nintendo Pokémon and Pokémon character names are trademarks of Nintendo. The Google Maps Pin is a trademark of Google Inc. and the trade dress in the product design is a trademark of Google Inc. under license to The Pokémon Company. Other trademarks are the property of their respective owners.
+[Privacy Policy](http://www.pokemon.com/us/privacy-policy/)
 
-The source icons belong to © Nintendo/Creatures Inc./GAME FREAK Inc.
+[PokemonGo-Bot](https://github.com/PokemonGoF/PokemonGo-Bot) is intended for academic purposes and should not be used to play the game *PokemonGo* as it violates the TOS and is unfair to the community. Use the bot **at your own risk**.
 
```

Comment:
Do not endorse anything for legal reasons. Use at your own risk for legal reasons. TOS statement.